app/models/procedure.rb

```diff
@@ -1,4 +1,3 @@
 class Procedure < ActiveRecord::Base
-    has_many :facilities_procedures
-    has_many :facilities, through: :facilities_procedures #might be facility_procedures
+    has_one :facility, :foreign_key => "facility_id"
```

Comment:
use the syntax `foreign_key: :facility_id`

Reply:
Changed to hash syntax in item.rb and procedure.rb.